webapp/src/components/Profile.tsx

```diff
@@ -1,7 +1,122 @@
 import React from 'react';
+import {
+  Container,
+  Grid,
+  Avatar,
+  Tooltip,
+  Divider,
+  IconButton,
+  Typography,
+} from '@mui/material';
+
+import GitHubIcon from '@mui/icons-material/GitHub';
+import LinkedInIcon from '@mui/icons-material/LinkedIn';
+import LanguageIcon from '@mui/icons-material/Language';
+import EmailIcon from '@mui/icons-material/Email';
+
+const user = {
```

Comment:
I see we are hard coding the user data here for now! I think this is good to set up this Profile component, and we can remove it later once we figure out the database information.